hadoop-ozone/dist/src/main/license/jar-report.txt

```diff
@@ -242,23 +243,25 @@ share/ozone/lib/ratis-thirdparty-misc.jar
 share/ozone/lib/ratis-tools.jar
 share/ozone/lib/re2j.jar
 share/ozone/lib/reflections.jar
-share/ozone/lib/rocksdb-checkpoint-differ.jar
 share/ozone/lib/reload4j.jar
+share/ozone/lib/rocksdb-checkpoint-differ.jar
 share/ozone/lib/rocksdbjni.jar
+share/ozone/lib/simpleclient.jar
 share/ozone/lib/simpleclient_common.jar
 share/ozone/lib/simpleclient_dropwizard.jar
-share/ozone/lib/simpleclient.jar
 share/ozone/lib/slf4j-api.jar
 share/ozone/lib/slf4j-reload4j.jar
 share/ozone/lib/snakeyaml.jar
 share/ozone/lib/snappy-java.jar
+share/ozone/lib/solr-solrj.jar
```

Comment:
solr dependency can be removed here

Reply:
solr dependency and other jar are present extra, not present earlier, can revert this file